lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -164,8 +185,15 @@ export class PublicClientApplication {
      * To acquire only idToken, please pass clientId as the only scope in the Authentication Parameters
      * @returns {Promise.<TokenResponse>} - a promise that is fulfilled when this function has completed, or rejected if an error was raised. Returns the {@link AuthResponse} object
      */
-    acquireTokenPopup(request: AuthenticationParameters): Promise<TokenResponse> {
-        throw new Error("Method not implemented.");
+    async acquireTokenPopup(request: AuthenticationParameters): Promise<TokenResponse> {
+        if (this.interactionInProgress()) {
+            throw BrowserAuthError.createInteractionInProgressError();
+        }
+        const interactionHandler = new PopupHandler(this.authModule, this.browserStorage);
+        const navigateUrl = await this.authModule.createAcquireTokenUrl(request);
+        const popupWindow = interactionHandler.showUI(navigateUrl);
+        const hash = await interactionHandler.monitorWindowForHash(popupWindow, this.config.system.loadFrameTimeout, navigateUrl);
+        return interactionHandler.handleCodeResponse(hash);
```

Comment:
We can do this post the alpha release too if needed, but both popup APIs are exactly the same. Can we have a PopUp helper?

Reply:
I can do this as part of a pre alpha PR.

Reply:
done.